api-docs/overview-content/aggregate_meter_groups.md

```diff
@@ -0,0 +1,7 @@
+# Aggregate Meter Groups
```

Comment:
this file is not being referenced on the api-docs.
This means there is no way to view this. I don't remember adding this to be honest. Where should it go under? I think it fits under 'distribution generation groups'